modules/schema/src/main/resources/lucuma/odb/graphql/OdbSchema.graphql

```diff
@@ -6518,6 +6518,57 @@ type CreateGroupResult {
   group: Group!
 }
 
+  """
+  The guide star(s) and related information
+  """
+type GuideEnvironment {
+  """
+  The position angle
+  """
+  posAngle: Angle!
```

Comment:
This position angle is determined by the guide star that is selected and at the given time. A different guide star could have a different position angle, so it seems to make sense here. Also, the position angle was included in the guide environment in an earlier discussion.

Reply:
It depends on the position angle constraint.  The PI can constrain the position angle to θ and then the AGS algorithm doesn't modify it.  Or the PI can set it to the parallactic angle and again the AGS algorithm doesn't change it.

Reply:
I get the relation between guide star and position angle so maybe this is the correct place to keep up with it.  Even before a target is selected or a guide star is found though, an observation can have a position angle.  If we want to know the position angle of the observation we'll need to look in the `GuideEnvironment` but it may be unset?

Reply:
I don't know. It seems to me that you can get all the information about the position angle you can get without knowing the time and guide star from the position angle constraint. If it is constrained to 0 degrees, then that will be the angle of the observation and it won't be changed here. However, if it is unconstrained, then won't the position angle of the observation depend on the time and the guide star selected and so can't be known until we select a guide star?